src/pages/MainPage/components/PostSection.vue

```diff
@@ -0,0 +1,26 @@
+<script setup>
+import PostCard from '@/components/PostCard.vue';
+import StatusBadge from '@/components/StatusBadge.vue';
+
+defineProps({
+  title: String,
+  badgeText: String,
+  badgeStatus: String,
+  posts: {
+    type: Array,
+    default: () => [],
```

Comment:
배열이니까 default는 이렇게 두는 게 더 괜찮을 것 같습니다
```suggestion
    default: [],
```